src/js/view/week/timeGrid.js

```diff
@@ -93,6 +93,21 @@ function getHoursLabels(opt, hasHourMarker, timezoneOffset, styles) {
         };
     });
 }
+
+/**
+ * Returns timezone offset from timezone object
+ * @param {object} timezoneObj - timezone object in options.timzones
+ * @param {number} timestamp - timestamp
+ * @returns {number} timezoneOffset - timezone offset
+ */
+function getTimezoneOffsetByTimezoneOption(timezoneObj, timestamp) {
+    var timezoneOffset = util.isNumber(timezoneObj.timezoneOffset) ?
```

Comment:
I think `timezoneObj.timezone` should have first priority than `timezoneOffset`.